Gum/Controls/ColorDisplay.xaml.cs

```diff
@@ -154,12 +155,17 @@ private void HandlePropertyChange(object sender, PropertyChangedEventArgs e)
             }
         }
 
-        private void HandleColorChange(object sender, RoutedPropertyChangedEventArgs<Color?> e)
-        {
+        private void HandleColorChange(object sender, RoutedEventArgs e) {
+            if (!(e is ColorRoutedEventArgs colorArgs)) return;
+
+            var prevColor = (Microsoft.Xna.Framework.Color)InstanceMember.Value;
+
+            var colorPack = (uint)(colorArgs.Color.R | (colorArgs.Color.G << 8) | (colorArgs.Color.B << 16) | (colorArgs.Color.A << 24));

```

Comment:
This check is necessary because `Pixi.ColorPicker` seems to fire the `ColorChanged` event multiple times per change (around 8 if I'm not mistaken), which makes the picker painful to use. There might be a better way to solve this, but I'm no WPF developer so I'd appreciate any feedback there.

Reply:
I'm not too familiar with this picker myself, but if this restricts the spamming of events, then I think it's a good solution. I'll approve it and see how it works.